exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/WorkspaceConfig.java

```diff
@@ -25,23 +25,27 @@
  *  - location which is a path.
  *  - writable flag to indicate whether the location supports creating new tables.
  *  - default storage format for new tables created in this workspace.
+ *  - physicalLocation is full path to workspace
  */
 @JsonIgnoreProperties(value = {"storageformat"})
 public class WorkspaceConfig {
 
   /** Default workspace is a root directory which supports read, but not write. */
-  public static final WorkspaceConfig DEFAULT = new WorkspaceConfig("/", false, null);
+  public static final WorkspaceConfig DEFAULT = new WorkspaceConfig("/", false, null, null);
 
   private final String location;
   private final boolean writable;
   private final String defaultInputFormat;
+  private final String physicalLocation;
 
   public WorkspaceConfig(@JsonProperty("location") String location,
                          @JsonProperty("writable") boolean writable,
-                         @JsonProperty("defaultInputFormat") String defaultInputFormat) {
+                         @JsonProperty("defaultInputFormat") String defaultInputFormat,
+                         @JsonProperty("physicalLocation") String physicalLocation) {
     this.location = location;
     this.writable = writable;
     this.defaultInputFormat = defaultInputFormat;
+    this.physicalLocation = physicalLocation;
```

Comment:
WorkspaceConfig already has "location". Why do you prefer adding a new "physicalLocation"? Is it possible to use "location" in stead?